rules/github.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+const execa = require('execa');
+const got = require('got');
+const gh = require('github-url-to-object');
+const rule = require('unified-lint-rule');
+
+module.exports = rule('remark-lint:awesome/github', async (ast, file) => {
+	const {dirname} = file;
+
+	try {
+		const remoteUrl = await execa.stdout('git', [
+			'config',
+			'--get',
+			'remote.origin.url'
+		], {
+			cwd: dirname
+		});
+		const githubUrls = gh(remoteUrl);
+		if (!githubUrls) {
+			file.message('Repository should be on GitHub');
+			return;
+		}
+
+		const headers = {
+			Accept: 'application/vnd.github.mercy-preview+json',
+			'User-Agent': 'awesome-lint'
+		};
+		if (process.env.github_token) {
+			headers.Authorization = `token ${process.env.github_token}`;
+		}
+
+		const res = await got.get(githubUrls.api_url, {
+			headers,
+			json: true
+		});
+
+		const data = res.body;
+		if (!data.description) {
+			file.message('The repository should have a description');
+		}
+
+		if (!data.topics.includes('awesome')) {
+			file.message('The repository should have "awesome" as GitHub topics');
+		}
+
+		if (!data.topics.includes('awesome-list')) {
+			file.message('The repository should have "awesome-list" as GitHub topics');
```

Comment:
Let's change the English here to read:

`The repository should have "awesome-list" as a GitHub topic`

Same thing with the other message above

Reply:
Nice catch!